src/middlewared/middlewared/plugins/datastore/write.py

```diff
@@ -20,6 +40,7 @@ class Config:
             'options',
             Bool('ha_sync', default=True),
             Str('prefix', default=''),
+            Bool('batch_operation', default=False),
```

Comment:
Can we give it a name that truly reflects its purpose? E.g. `Bool('send_events', default=True),`